frontend/src/components/activePage/TableInstance/RowInstanceActions/RowInstanceActionsPersistent.tsx

```diff
@@ -7,57 +7,81 @@ import {
   PlayCircleOutlined,
 } from '@ant-design/icons';
 import { Instance } from '../../../../utils';
+import { useApplyInstanceMutation } from '../../../../generated-types';
+import { setInstanceRunning } from '../../ActiveUtils';
 
 export interface IRowInstanceActionsPersistentProps {
   extended: boolean;
   instance: Instance;
-  startInstance: (idInstance: string, idTemplate: string) => void;
-  stopInstance: (idInstance: string, idTemplate: string) => void;
 }
 
 const RowInstanceActionsPersistent: FC<IRowInstanceActionsPersistentProps> = ({
   ...props
 }) => {
-  const { extended, instance, startInstance, stopInstance } = props;
+  const { extended, instance } = props;
 
-  const { status, idTemplate, name } = instance;
+  const { status, running } = instance;
 
   const font22px = { fontSize: '22px' };
 
+  const [disabled, setDisabled] = useState(false);
+
+  const [applyInstanceMutation] = useApplyInstanceMutation();
+
+  const setInstanceStatus = async () => {
+    if (!disabled) {
+      setDisabled(true);
+      try {
+        const result = await setInstanceRunning(
+          running!,
+          instance,
+          applyInstanceMutation
+        );
+        if (result) setTimeout(() => setDisabled(false), 400);
```

Comment:
```suggestion
        if (result) setTimeout(setDisabled, 400, false);
```
Use this to increase prowess 🤺